### ISSUE TITLE
Duplicate a request

### ISSUE THREAD

**Issue by leearmstrong** [eBay/fabio#261]
Is it possible to have HTTP requests duplicated to say an R&D server?

The original server(s) handle the response path but the R&D server response is not sent back to the client.

**Comment by magiconair** [comment]
No, that isn't possible right now. You could use [Traffic Shaping](https://github.com/eBay/fabio/wiki/Features#traffic-shaping) to send a small amount of traffic to another server but this isn't duplicating the request. 

Could you explain your use case a bit more? My guess is testing but then your R&D server would need to be able to handle production load. Are there other proxies that support this?
